tests/core/eth-module/test_transactions.py

```diff
@@ -18,7 +18,6 @@
         pytest.param(
             lambda web3: 999999999999,
             False,
-            marks=pytest.mark.xfail,
```

Comment:
Following the approach on short-circuiting if `chainId` is `None`, this line should be left in.